cmake/ccf.cmake

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+if((NOT "sgx" IN_LIST TARGET) AND (NOT "virtual" IN_LIST TARGET))
+  message(FATAL_ERROR "Variable list 'TARGET' must be defined and include a valid target")
```

Comment:
Perhaps give an example of a valid target in the error message?

Reply:
Sure, done!